File: fixtures/magic-numbers/consts.py
# fixtures/magic-numbers/consts.py
MAX = 5
MIN_TEMP = -40


def check_threshold(x):
    if x > 42:
        return True
    return False


def check_lower_bound(x):
    return x < -273


def local_scope():
    local_not_module_level = 99
    return local_not_module_level

File: fixtures/magic-numbers/consts.ts
// fixtures/magic-numbers/consts.ts
const MAX_RETRIES = 5;
const MIN_TEMP = -40;

export function checkThreshold(x: number): boolean {
  if (x > 42) {
    return true;
  }
  return false;
}

export function checkLowerBound(x: number): boolean {
  return x < -273;
}

interface RunOutput {
  version: 6;
}

export function allowlistedValues(x: number): number {
  if (x === 0) return 0;
  if (x === 1) return 1;
  return -1;
}

export function scheduleThing(fn: () => void): void {
  setTimeout(fn, 3000);
}

function localScope(): number {
  const LOCAL_NOT_MODULE_LEVEL = 99;
  return LOCAL_NOT_MODULE_LEVEL;
}
